vizro-core/docs/pages/user_guides/parameters.md

```diff
@@ -0,0 +1,176 @@
+                        color="species",
+                        size="petal_length",
+                        color_discrete_map={"setosa": "#00b4ff", "versicolor": "#ff9222"},
+                    ),
+                ),
+                vm.Graph(
+                    id="bar_chart",
+                    figure=px.bar(
+                        iris,
+                        x="sepal_width",
+                        y="sepal_length",
+                        color="species",
+                        color_discrete_map={"setosa": "#00b4ff", "versicolor": "#ff9222"},
+                    ),
+                ),
+            ],
+            controls=[
+                vm.Parameter(
+                    targets=["scatter_chart.color_discrete_map.virginica", "bar_chart.color_discrete_map.virginica"],
+                    selector=vm.Dropdown(
+                        options=["#ff5267", "#3949ab"],
+                        multi=False,
+                        value="#3949ab",
+                    ),
+                ),
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: scatter
+                  data_frame: iris
+                  x: sepal_width
+                  y: sepal_length
+                  size: petal_length
+                  color: species
+                  color_discrete_map: {"setosa": "#00b4ff", "versicolor": "#ff9222"}
+                id: scatter_chart
+                type: graph
+              - figure:
+                  _target_: bar
+                  data_frame: iris
+                  x: sepal_width
+                  y: sepal_length
+                  color: species
+                  color_discrete_map: {"setosa": "#00b4ff", "versicolor": "#ff9222"}
+                id: bar_chart
+                type: graph
+            controls:
+              - selector:
+                  options: ["#ff5267", "#3949ab"]
+                  value: #3949ab
+                  multi: False
+                  type: dropdown
+                targets: ["scatter_chart.color_discrete_map.virginica", "bar_chart.color_discrete_map.virginica"]
+                type: parameter
+            title: My first page
+        ```
+    === "Result"
+        [![Nested]][Nested]
+
+    [Nested]: ../../assets/user_guides/control/control5.png
+
+In the above example, the object passed to the function argument `color_discrete_map` is a dictionary which maps the different flower species to fixed colors (eg. `{"virginica":"blue"}`). In this case we do not want to change the entire dictionary, but only the value `blue`. We do this by specifying a target as `scatter.color_discrete_map.virginica`.
```

Comment:
```suggestion
In the above example, the object passed to the function argument `color_discrete_map` is a dictionary which maps the different flower species to fixed colors (eg. `{"virginica":"blue"}`). In this case, only the value `blue` should be changed instead of the entire dictionary. This can be achieved by specifying a target as `scatter.color_discrete_map.virginica`.
```